### PR TITLE
Create genome-interval-coordinates.md

### PR DESCRIPTION
**Pull request by mbaudis**
This is the documentation provided by @andrewyatz for issue #3. Please review ...

cc @jmarshall @reece

### CODE REVIEW

**Comment by mbaudis** [comment]
Ping @jmarshall @reece ... Want to go ahead here, please :-)

**Comment by andrewyatz** [comment]
We raised this just now in the LSG call now

**Comment by reece** [comment]
Looks good to me! It's handy to have a summary of conventions in one place. Okay to merge.

**Comment by jmarshall** [comment]
I am working on some additions. Let's not merge this just yet.

File: pages/_formats/genome-interval-coordinates.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+When humans refer to a range of residues within a sequence, the most common convention is to use an interval of ordinal residue positions in the sequence i.e. start counting residues from 1. This system is also referred to as "1-start, fully-closed", biological coordinates and "Ensembl style". While natural for humans, this convention has several shortcomings for data modelling and programming. GA4GH prefers the use of interbase or "0-based, half-open" coordinates (also known as Chado or "UCSC style") and strongly advises that all future products prefer their use for future products unless the product visually displays data to a human. Interbase coordinates refer to the zero-width points before and after residues. An interval of interbase coordinates permits referring to any span, including an empty span, before, within, or after a sequence.
+
+#### The Interbase Coordinate System
+
+While interbase is numerically equivalent to "0-start, fully-closed" they are semantically different. Interbase does not refer to residues and therefore can model events occurring between residues, the start and end of a sequence. For non-circular sequences the following holds true.
```
Comment (jmarshall) [inline pages/_formats/genome-interval-coordinates.md]: What it's numerically equivalent to is 0-start **half-open**.

Comment (mbaudis) [reply pages/_formats/genome-interval-coordinates.md]: Fixed. Also, 2 pages now; one more general/recommendation; one (@jmarshall 's) with more details to interbase use etc.

IMO this page here could be condensed a bit now.

**Comment by mbaudis** [comment]
@andrewyatz @jmarshall @reece I'm merging this now; however, the page(s) still have a _DRAFT_ label, and are open for later edits/trimming/refinements.

IMO it is important to have a documentation/standards use case __live__, where we can also work on our procedures.

**Comment by jmarshall** [comment]
As @andrewyatz is out of the office until the 25th, I had not yet reviewed this other than to note that one obvious thinko.

Further review comments:

Assuming the goal here is simply to recommend use of a 0-based half-open / interbase coordinate system in APIs, the text here adds further requirements that are a distraction from this goal. For example:

* `uint64`
* emphasis on interbase in particular
* non-circular

None of these are fundamental properties of the coordinate system being promoted, so they distract from the core message.

The list of properties is also a bit of a distraction, and it's not made clear what you mean by reverse start / reverse end. The formula for reverse end is off by one.

Re the table of existing products, SAM/BAM/CRAM/VCF/BCF primarily represent positions+various lengths, not intervals per se. So it's mostly not accurate to say they support either 0-start half-open or 1-start fully-closed intervals. (Were there interval fields that you had in mind?) OTOH SAM and VCF use 1-based positions (as they are human readable) while BAM, CRAM, and BCF use 0-based postions (as they are binary machine-orientated formats).

In fact the htsget, refget, and Beacon specifications all describe their start/end parameters in 0-based half-open terms. But fundamentally for a protocol Interbase vs. 0-start half-open is a distinction without a difference: the protocol has start/end fields, and clients and implementers are free to think of them as 0-start half-open or interbase as they wish.

My proposal for dealing with these review comments is to merge the two pages: see #3 and PR #7.

**Pull request merged**